### DISCUSSION TITLE
Multiple Audio Tracks?

### DISCUSSION THREAD

**Discussion by AlexAtPSU** [Zulko/moviepy#1404] [General]
I'm working with a video file that has multiple audio tracks for different languages. 
I might have missed it but I couldn't find a way on the documentations to choose which audio to grab from the video.
Is there a way to choose which audio when giving it the video file or when creating a AudioFileClip from the video?

**Comment by tburrows13** [comment]
Hi @AlexAtPSU!
There isn't currently a way to do that in MoviePy. If you know FFmpeg, then you could pass mess around with the ffmpeg command [here](https://github.com/Zulko/moviepy/blob/master/moviepy/audio/io/readers.py#L96) and add your channel selection there.
